continuous_integration/install.sh

```diff
@@ -73,6 +73,9 @@ if [ -z "$NUMPY_VERSION" ]; then
     # setup.cfg so doing it the hacky way ...
     cat setup.cfg | grep -v 'with-doctest=' > setup.cfg.new
     mv setup.cfg{.new,}
+else
```

Comment:
Actually maybe it was intentional: we wanted to make sure that there was no hard dependency on memory_profiler. Kind of rings a bell, sorry I forgot about that. Maybe add a comment about this.